aws.js

```diff
@@ -79,85 +83,82 @@ new (function() {
 			Text: txt,
 			VoiceId: voiceId,
 		};
-		
-		polly.synthesizeSpeech(params, function(err, data) {
+
+		polly.synthesizeSpeech(params, function (err, data) {
 			if (err)
 				console.log(err, err.stack);
 			else {
 				var uInt8Array = new Uint8Array(data.AudioStream);
 				var arrayBuffer = uInt8Array.buffer;
 				var blob = new Blob([arrayBuffer]);
 				var url = URL.createObjectURL(blob);
-				
+
 				playAudioFromUrl(url, callback);
 			}
-		});		
+		});
 	}
-	function comparethem(bucketsource,bucketinput) {
+
+	function comparethem(bucketsource, bucketinput) {
 		var comparams = {
-                       SimilarityThreshold: 20,
-                       SourceImage: {
-                       S3Object: {
-                       Bucket: bucketsource,
-                       Name: "IMG_20180823_183435.jpg"
-                       }
-                      },
-                       TargetImage: {
-                       S3Object: {
-                       Bucket: bucketinput,
-                       Name: "IMG_20190106_204146.jpg"
-                        }
-                       }
+			SimilarityThreshold: 20,
+			SourceImage: {
+				S3Object: {
+					Bucket: bucketsource,
+					Name: "IMG_20180823_183435.jpg"
+				}
+			},
+			TargetImage: {
+				S3Object: {
+					Bucket: bucketinput,
+					Name: "IMG_20190106_204146.jpg"
+				}
+			}
 		};
 		prompt("Hello")
-		rekognition.compareFaces(comparams, function(err, data) {
```

Comment:
@aswinkumar2019. you are not integrating rekognition. You are just creating variable called rekognition